### PR TITLE
Added basic main menu structure

### PR DESCRIPTION
**Pull request by frnzee**
- added main menu prefab and class, implemented it in a way of the project structure

### CODE REVIEW

File: Assets/Scripts/Scenarios/LaunchMainMenuScenario.cs [hunk 1]
```diff
@@ -25,11 +25,11 @@ public void Construct(IUIRoot uiRoot, LoadingService loadingService, LevelLoadin
 
         protected void Start()
         {
-            //var view = _uiRoot.Open<StartSplashScreen>();
-            //_uiRoot.Open<MainMenu>();
-            //view.Closed += OnViewClosed;
-            var view = _uiRoot.Open<StartSplashScreen>();
-            view.Closed += OnSplashScreenClosed;
+            var mainMenuView = _uiRoot.Open<MainMenu>();
```
Comment (ega-ega) [inline Assets/Scripts/Scenarios/LaunchMainMenuScenario.cs]: This Behaviour must be invoce after start splash screen closed/ Pls move to anpther method

File: Assets/Scripts/UI/Roots/UIRoot.cs [hunk 2]
```diff
@@ -28,7 +28,7 @@ public void Construct(UIBuilder uiBuilder)
                 return default;
             }
             
-            if(_viewStack.Any())
+            if (_viewStack.Any())
```
Comment (ega-ega) [inline Assets/Scripts/UI/Roots/UIRoot.cs]: This changes not requaried for you task pls. revert this file

File: Assets/Scripts/Scenarios/LaunchMainMenuScenario.cs [hunk 3]
```diff
@@ -25,11 +25,11 @@ public void Construct(IUIRoot uiRoot, LoadingService loadingService, LevelLoadin
...
+            mainMenuView.Closed += OnMainMenuClosed;
```
Comment (ega-ega) [inline Assets/Scripts/Scenarios/LaunchMainMenuScenario.cs]: Subscibe to thi method not neccesary. Pls remove.

File: Assets/Scripts/Scenarios/LaunchMainMenuScenario.cs [hunk 4]
```diff
@@ -43,9 +43,15 @@ private async void  WaitAndLoadLevel()
         private void OnSplashScreenClosed(IView view)
         {
             view.Closed -= OnSplashScreenClosed;
-            //_uiRoot.Open<MainMenu>();
+            _uiRoot.Open<MainMenu>();
+        }
+        
+        private void OnMainMenuClosed(IView view)
```
Comment (ega-ega) [inline Assets/Scripts/Scenarios/LaunchMainMenuScenario.cs]: This behaviour must be Main menu. Pls move this method and method WaitAndLoadLevel to main menu class

File: Assets/Scripts/Scenarios/LaunchMainMenuScenario.cs [hunk 5]
```diff
@@ -43,9 +43,15 @@ private async void  WaitAndLoadLevel()
...
+        {
+            view.Closed -= OnMainMenuClosed;
             WaitAndLoadLevel();
             Destroy(gameObject);
```
Comment (ega-ega) [inline Assets/Scripts/Scenarios/LaunchMainMenuScenario.cs:27]: Move Destroy to OnSplashScreenClosed

File: Assets/Prefabs/UI/MainMenu/MainMenu.prefab
Comment (ega-ega) [inline Assets/Prefabs/UI/MainMenu/MainMenu.prefab:1]: Move this prefab to Popup folder and remove Main Menu folder

**Review by ega-ega — CHANGES_REQUESTED** [review]

File: Assets/Scripts/Scenarios/LaunchMainMenuScenario.cs (hunk 1 above)
Comment (frnzee) [reply Assets/Scripts/Scenarios/LaunchMainMenuScenario.cs]: Done

File: Assets/Scripts/UI/Roots/UIRoot.cs (hunk 2 above)
Comment (frnzee) [reply Assets/Scripts/UI/Roots/UIRoot.cs]: Reverted

File: Assets/Scripts/Scenarios/LaunchMainMenuScenario.cs (hunk 3 above)
Comment (frnzee) [reply Assets/Scripts/Scenarios/LaunchMainMenuScenario.cs]: Removed

File: Assets/Scripts/Scenarios/LaunchMainMenuScenario.cs (hunk 4 above)
Comment (frnzee) [reply Assets/Scripts/Scenarios/LaunchMainMenuScenario.cs]: Done

File: Assets/Scripts/Scenarios/LaunchMainMenuScenario.cs (hunk 5 above)
Comment (frnzee) [reply Assets/Scripts/Scenarios/LaunchMainMenuScenario.cs:27]: Done

**Review by ega-ega — APPROVED** [review]

**Pull request merged**